.github/workflows/mainline_integration_test.yml

```diff
@@ -1,6 +1,8 @@
 name: Mainline Merge Integration Tests
 
 on:
+  schedule:
+    - cron: '0 */4 * * *'
```

Comment:
Chatted offline, 

Why 4 ? Because worker agent side is running every 4 hours too.